src/python/pcmdi/scripts/pcmdi_metrics_driver.py

```diff
@@ -108,12 +102,14 @@ def applyCustomKeys(O,custom_dict,var):
     sftlf[model_version]["md5"]=None
 if parameters.targetGrid == "2.5x2.5":
   tGrid = cdms2.createUniformGrid(-88.875,72,2.5,0,144,2.5)
-else:
+
+if parameters.targetGrid == "1.0x1.0":
+  tGrid = cdms2.createUniformGrid(-89,179,1,0,360,1)
+
+if parameters.targetGrid not in ['2.5x2.5','1.0x1.0']:
   tGrid = parameters.targetGrid
 
-sft = cdutil.generateLandSeaMask(tGrid)
-sft[:]=sft.filled(1.)*100.
-sftlf["targetGrid"] = sft
+sftlf["targetGrid"] = cdutil.generateLandSeaMask(tGrid)*100.
```

Comment:
why removing the filled(1.) I think it is needed in some case land comes back as missing